### PR TITLE
Add type and status labels

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: views/includes/block-content.pug [hunk 1]
```diff
@@ -226,6 +239,10 @@ div(class="tab-content")
 									span  
 									a(data-toggle="tooltip", title=(coinConfig.name + " Fun! See transaction for details"))
 										i(class="fas fa-certificate text-primary")
+								
+								span(class="rounded text-white px-2 mr-2 float-right bg-status") PENDING
```
Comment (mateuszpmroz) [inline views/includes/block-content.pug:243]: include ./status-labels.pug

File: views/address.pug [hunk 2]
```diff
@@ -345,6 +345,10 @@ block content
 														else
 															span(class="text-danger") Unconfirmed
 
+													div(class="text-md-right")
+														span(class="rounded text-white px-2 mr-2 float-right bg-status") PENDING
```
Comment (mateuszpmroz) [inline views/address.pug:349]: include ./status-labels.pug

File: views/unconfirmed-transactions.pug [hunk 3]
```diff
@@ -30,6 +30,9 @@ block content
 								strong ##{(txIndex + offset).toLocaleString()}
 								span  &ndash; 
 								a(href=("/tx/" + tx.txid)) #{tx.txid}
+
+								span(class="rounded text-white px-2 mr-2 float-right bg-status") PENDING
```
Comment (mateuszpmroz) [inline views/unconfirmed-transactions.pug:34]: include ./status-labels.pug

**Review by harzo — APPROVED** [review]

**Comment by mateuszpmroz** [comment]
Cannot give approve because PR was created by me, but approve from me :+1:

**Pull request merged**